docs/user-guide/tls-configuration.md

```diff
@@ -0,0 +1,85 @@
+# Customizing Native TLS
+
+Zowe's servers have built-in TLS support to enable HTTPS connections.
```

Comment:
If you want to break up these sentences into two paragraphs, you need a blank line here to create a line break.

Reply:
Done thanks!